client/components/deposits-information/test/index.js

```diff
@@ -67,7 +72,13 @@ const mockOverviews = ( currencies = null, account = null ) => {
 
 useInstantDeposit.mockReturnValue( {
 	deposit: undefined,
-	isLoading: false,
+	inProgress: false,
```

Comment:
I noticed this `useInstantDeposit` `isLoading` prop should be named `inProgress`, as per naming in https://github.com/Automattic/woocommerce-payments/blob/884c7fdf6f997f4c82c3c1d808784b507e09d297/client/data/deposits/hooks.ts#L187